### PR TITLE
CVE-2019-14855 init.

### PR DESCRIPTION
**Pull request by guitos**

### CODE REVIEW

File: 2019/14xxx/CVE-2019-14855.json [hunk 1]
```diff
@@ -0,0 +1,74 @@
...
+                "name": "https://bugzilla.redhat.com/show_bug.cgi?id=CVE-2019-14855",
+                "refsource": "CONFIRM"
+            }
+        ]
+    },
+    "description": {
+        "description_data": [
+            {
+                "lang": "eng",
+                "value": "A flaw was found in the way certificate signatures could be forged using collisions found in the SHA-1 algorithm. An attacker could use this weakness to create forged certificate signatures. This issue affects GnuPG versions before 2.0."
```
Comment (iamleot) [inline 2019/14xxx/CVE-2019-14855.json]: The `2.0` does not match with version used in version data and with reality.  I think that the this issue affects GnuPG versions before 2.2.18.

File: 2019/14xxx/CVE-2019-14855.json [hunk 2]
```diff
@@ -0,0 +1,74 @@
...
+        "reference_data": [
+            {
+                "url": "https://rwc.iacr.org/2020/slides/Leurent.pdf"
+            },
+            {
+                "url": "https://bugzilla.redhat.com/show_bug.cgi?id=CVE-2019-14855",
+                "name": "https://bugzilla.redhat.com/show_bug.cgi?id=CVE-2019-14855",
+                "refsource": "CONFIRM"
+            }
+        ]
```
Comment (iamleot) [inline 2019/14xxx/CVE-2019-14855.json]: Please add upstream references:

 https://lists.gnupg.org/pipermail/gnupg-announce/2019q4/000442.html
 https://dev.gnupg.org/T4755

Comment (guitos) [reply 2019/14xxx/CVE-2019-14855.json]: Fix version information and add upstream references.
Thank you for your suggestions!

**Comment by iamleot** [comment]
Guilherme de Almeida Suckevicz writes:
> [...]
> Fix version information and add upstream references.
> Thank you for your suggestions!

Thank you Guilherme for quickly updating them!

**Pull request merged**